test/fixtures/meek_fronted_requests.txt

```diff
@@ -0,0 +1 @@
+a0.awsstatic.com:d2zfqthxsdq309.cloudfront.net
```

Comment:
If possible, I'd like to have two-three entries here (in general, things supposed to run more than once are more thoroughly tested if run more than once; when running once, again in general, I always have the doubt because you never know whether the state is properly reset after the first run)